src/main/webapp/app/exercises/programming/shared/code-editor/instructions/code-editor-instructions.component.html

```diff
@@ -20,7 +20,9 @@ <h3 class="card-title">
         </h3>
         <fa-icon *ngIf="!collapsed" class="ms-auto clickable" [icon]="faChevronRight"></fa-icon>
     </div>
-    <ng-content *ngIf="!collapsed"></ng-content>
+    <div style="display: contents" [hidden]="collapsed">
+        <ng-content></ng-content>
+    </div>
```

Comment:
```suggestion
    <ng-container [hidden]="collapsed">
        <ng-content></ng-content>
    </ng-container>
```
Should solve the problem :)

Reply:
> Should solve the problem, or?

Tried this too but does not work as `ng-container` isn't an actual DOM element and hence `[hidden]` directive can't be used and style changes not injected..

```
 error NG8002: Can't bind to 'hidden' since it isn't a known property of 'ng-container'.
```